docs/rest-api-gateway-guide.md

```
@@ -0,0 +1,268 @@
+### Core Endpoints
+
+#### Tool Discovery
+```http
+GET /api/v1/tools
+```
+
+**Query Parameters**:
+- `category` - Filter by tool category (e.g., "AI Tools", "System", "Troubleshooting")
+- `tag` - Filter by a single tag (e.g., "kubernetes", "deployment", "analysis") 
+- `search` - Search tool names and descriptions
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)